### PR TITLE
docs: Add community guides

### PR DESCRIPTION
**Pull request by litleleprikon**
Add contribution guides and PR/Issue templates for better contribution experience.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,33 +23,9 @@ Configure triggers at `localhost:8080` using your browser.
 
 Other installation methods are available, see [documentation](https://moira.readthedocs.io/en/latest/installation/index.html).
 
-## Development
+## Contribution
```
Comment (borovskyav) [inline README.md]: Давай заодно немного в файле приберемся. Здесь, по моему мнению сейчас слишком много лишней и "устаревшей информации"
Устаревшая: я бы избавился от упоминания 2.0. 3 года уже 2.0 а мы кичимся этим.
Что бы еще поправил:
Я бы сейчас сделал 3 раздела в таком порядке:
Getting Started - так бы и оставил
Contribution - идеально переделал =)
Installation - скинул бы ссылку на https://moira.readthedocs.io/en/latest/installation/index.html

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    * [Git Style Guide](#git-style-guide)
+    * [Go Code Style Guide](#go-code-style-guide)
+    * [Testing Strategy](#testing-strategy)
+* [Development](#development)
+
+## How Can I Contribute?
+
+### Submitting an Issue
+
+If you find bugs or you want propose an idea that you think will be suitable for moira you can create an issue. The main
```
Comment (beevee) [inline .github/CONTRIBUTING.md]:
```suggestion
If you find bugs or you want propose an idea that you think will be suitable for Moira you can create an issue. The main 
```

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    * [Testing Strategy](#testing-strategy)
+* [Development](#development)
+
+## How Can I Contribute?
+
+### Submitting an Issue
+
+If you find bugs or you want propose an idea that you think will be suitable for moira you can create an issue. The main 
+issue tracker is https://github.com/moira-alert/moira/issues. Issues that belongs for moira backend and web are placed here. 
+Issues that are related to python client and ansible role should be placed in issue tracker for related repo.
```
Comment (beevee) [inline .github/CONTRIBUTING.md]: What about docs?

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+We have a templates for two types of issues: Feature Request and a Bug. If you want to submit a bug please be patient
```
Comment (beevee) [inline .github/CONTRIBUTING.md]:
```suggestion
We have templates for two types of issues: a Feature Request and a Bug. If you want to submit a bug please be patient 
```

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+### Submitting an Issue
+
+If you find bugs or you want propose an idea that you think will be suitable for moira you can create an issue. The main 
+issue tracker is https://github.com/moira-alert/moira/issues. Issues that belongs for moira backend and web are placed here. 
+Issues that are related to python client and ansible role should be placed in issue tracker for related repo.
+
+We have a templates for two types of issues: Feature Request and a Bug. If you want to submit a bug please be patient 
+and fill all required fields. This will help us to define the problem. 
+
+Try to follow this advices to fill a great issue:
```
Comment (beevee) [inline .github/CONTRIBUTING.md]:
```suggestion
Try to follow this advice to fill a great issue:
```

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+
+* Check that issue tracker do not contain issues similar to your.
```
Comment (beevee) [inline .github/CONTRIBUTING.md]:
```suggestion
* Check that issue tracker does not contain issues similar to your.
```

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+* Use a clear and descriptive title for the issue to identify the suggestion.
+* Provide a step-by-step description of the suggested enhancement in as many details as possible.
+* Provide specific examples to demonstrate the steps. Include copy/pasteable snippets which you use in those examples, as Markdown code blocks.
+* Describe the current behavior and explain which behavior you expected to see instead and why.
+* Include screenshots and animated GIFs which help you demonstrate the steps to reproduce issue.
+* Explain why this enhancement would be useful to most Moira users.
+* Specify which version of Moira you're using.
```
Comment (beevee) [inline .github/CONTRIBUTING.md]: Can we provide a link to a good example of issue descrtiption?

File: .github/CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+* In footer place an additional information about your commit. Specify issue number in footer with `Relates` or `Closes` label.
+
+Structure of commit:
+
+```
+<type>[optional scope]: <description>
+
+[optional body]
+
+[optional footer(s)]
```
Comment (beevee) [inline .github/CONTRIBUTING.md]: Provide an example of a good commit message?

File: .github/CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+* **perf:** A code change that improves performance
+* **refactor:** A code change that neither fixes a bug nor adds a feature
+* **style:** Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+* **test:** Adding missing tests or correcting existing tests
+
+### Go Code Style Guide
+
+We are using [GolangCI-Lint](https://github.com/golangci/golangci-lint) to check our code for following the code style.
+Check your code with `make lint` before you will submit it. Code with broken style will not pass the CI and will not be
+reviewed. In future core contributors team can change the rules for linter in project.
```
Comment (beevee) [inline .github/CONTRIBUTING.md]:
```suggestion
reviewed.
```

Of course we can change anything anytime. I don't see the point of reiterating this thought here. We don't strive to intimidate our reader, do we?

File: .github/CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+goconvey
+```
+
+For full local deployment of all services, including web, graphite and metrics relay (may be slow on first launch) use:
+
+```bash
+docker-compose up
+```
+
+Before push your changes don't forget about linter:
```
Comment (beevee) [inline .github/CONTRIBUTING.md]:
```suggestion
Before pushing your changes don't forget about linter:
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+
+# FEATURE
+
+## Summary
+
+Main idea
+
+### Component
+
+<!-- In which component this feature should be implemented -->
```
Comment (beevee) [inline .github/ISSUE_TEMPLATE/feature_request.md]: I don't think it's necessary. Even core developers often don't know the component scope of a task in advance. Most tasks reqiure touching more than one component. And I don't see how filling this section saves us any time.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 12]
```diff
@@ -0,0 +1,5 @@
+## Component
```
Comment (beevee) [inline .github/PULL_REQUEST_TEMPLATE.md]: Do we need this? Maybe labels with component names are more suitable?

File: README.md (hunk 1 above)
Comment (litleleprikon) [reply README.md]: Fixed

File: .github/CONTRIBUTING.md (hunk 8 above)
Comment (litleleprikon) [reply .github/CONTRIBUTING.md]: Provided

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 12 above)
Comment (litleleprikon) [reply .github/PULL_REQUEST_TEMPLATE.md]: Removed

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 11 above)
Comment (litleleprikon) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Removed

File: .github/CONTRIBUTING.md (hunk 7 above)
Comment (litleleprikon) [reply .github/CONTRIBUTING.md]: I am not sure in this case. The reason is that we have a lot of perfectly documented issues but those issues do not follow templates introduced in this PR. Probably the solution here will be to edit existent issue to follow template. I believe that this two issues will be a good candidates.

* https://github.com/moira-alert/moira/issues/481
* https://github.com/moira-alert/moira/issues/457

Comment (beevee) [reply .github/CONTRIBUTING.md]: Maybe we can leave this for now and add a link later, when we have a new issue that follows the template.

File: .github/CONTRIBUTING.md [hunk 13]
```diff
@@ -0,0 +1,158 @@
...
+    - [Go Code Style Guide](#go-code-style-guide)
+    - [Testing Strategy](#testing-strategy)
+  - [Development](#development)
+
+## How Can I Contribute
+
+### Submitting an Issue
+
+If you find bugs or you want propose an idea that you think will be suitable for Moira you can create an issue. The main 
+issue tracker is https://github.com/moira-alert/moira/issues. Issues that belongs for moira backend and web are placed here.
```
Comment (beevee) [inline .github/CONTRIBUTING.md]:
```suggestion
issue tracker is https://github.com/moira-alert/moira/issues. Issues that belong to moira backend and web are placed here. 
```

File: .github/CONTRIBUTING.md (hunk 7 above)
Comment (litleleprikon) [reply .github/CONTRIBUTING.md]: Let's do this

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (beevee) [reply .github/CONTRIBUTING.md]: Still nothing about docs :-)

Comment (litleleprikon) [reply .github/CONTRIBUTING.md]: I added a whole new section about docs. I do not understand what you mean here :(

Comment (beevee) [reply .github/CONTRIBUTING.md]: 1. Issues that belong to backend and web are placed here.
2. Issues that are related to python client and ansible role should be placed there.

What about issues that belong to docs?

Comment (beevee) [reply .github/CONTRIBUTING.md]: Also, `belongs for` → `belong to`.

Comment (litleleprikon) [reply .github/CONTRIBUTING.md]: I added a couple of words about docs and also placed couple of links for issue trackers.

**Pull request merged**